### DISCUSSION TITLE
firo-client wallet

### DISCUSSION THREAD

**Discussion by SaeeD2002DeeaS** [LedgerHQ/wallet-api#108] [General]
Good day to all,
I have issue for my wallet I lose passphrase and seed phrase, however I want recovered passphrase to lock wallet.
I heard and watch multi video how to recovered passwords but I tried from (wallet.dat) by hashcat  but I don't get any result 
so any one have a way to recovered passphrase.
Thanks

**Comment by adrienlacombe** [comment]
This is not the right place to post this, sorry.
